admin/tests/integration/components/certifications/issue-report_test.js

```diff
@@ -33,18 +38,40 @@ module('Integration | Component | certifications/issue-report', function (hooks)
   });
 
   module('when the issue has not been resolved yet', function () {
-    test('it should display resolve button', async function (assert) {
+    test('it should display resolve button if current user has access to certification actions scope', async function (assert) {
```

Comment:
Suggestion (pas obligatoire)
Quand il y a un if/when dans un `it` c'est qu'il manque probablement un contexte
```suggestion
    module('when current user has access to certification actions scope', function() {
       test('it should display resolve button', async function (assert) {
```

Reply:
perso je commence à mettre dans un `when` quand j'ai au moins 2 `it` à y mettre dedans ... mais je peux quand même séparer si c'est plus simple à lire

Reply:
ok